include/api.php

```diff
@@ -1556,22 +1557,31 @@ function api_search($type)
 		}
 		$terms = DBA::select('term', ['oid'], $condition, []);
 		$itemIds = [];
-		while($term = DBA::fetch($terms)){ $itemIds[] = $term['oid']; }
+		while ( $term = DBA::fetch($terms) ) {
+			$itemIds[] = $term['oid'];
+		}
 		DBA::close($terms);
-		$condition = [$exclude_replies ? "`id` = `parent` AND " : ''];
-		$condition[0] .= empty($itemIds) ? '' : ' `id` IN ('.implode(", ", $itemIds).')' ;
 
+		if (empty($itemIds)) {
+			return api_format_data("statuses", $type, $data);
+		}
+
+		$tmpAr = ['`id` IN ('.implode(", ", $itemIds).')'];
```

Comment:
Code standards: Please wrap concatenation operators with spaces and use single quotes for strings.

Reply:
thank you for waste your time @MrPetovan ;
ps. i will search something php lint on my vim plugin later

Reply:
It never is wasted time.